frontend/src/components/social/logsPreview/LogPreviewBox.jsx

```diff
@@ -129,17 +134,34 @@ const FrameRow = styled.div`
 
 const ProfileWrapper = styled.div`
     aspect-ratio: 1;
-    min-width: 50px;
-    width: 50px;
+    ${(props) =>
+        props.$isMe
+            ? css`
+                  width: calc(
+                      (50% - 1.5em) * ${props.$isDesktop ? 0.55 : 0.50}
+                  );
+              `
+            : css`
+                  width: ${props.$isDesktop ? 55 : 50}%;
+              `};
+    max-width: 4.5em;
 
-    ${ifMobile} {
-        max-width: 50px;
+    ${ifMobile} ${ifTablet} {
```

Comment:
```suggestion
    ${ifTablet} {
```

`ifTablet`로 스타일을 지정하면 그 이하의 화면 크기에도 스타일이 적용됩니다.